### PR TITLE
Clarify guide-level explanation

### PR DESCRIPTION
**Pull request by CLEckhardt**
Makes the guide-level text more explicit and direct through adding structure, rewording, and adding clarifying formatting.

Previously, some of the text, particularly when describing states and constraints, was difficult to understand.

### CODE REVIEW

File: text/0025-purchase-order.md [hunk 1]
```diff
@@ -309,31 +308,30 @@ Beginning state. An organization with po::draft permissions created a version
...
-Cancelled state. A version is no longer relevant; no further edits may be
-introduced to the version.
+Cancelled state. A version is no longer relevant; it is no longer editable.
 
 ### Purchase Order Collaborative Sub-Workflow
 
 ![PO Collaborative Sub-Workflow](../images/0025-purchase-order-rfc/po_collaborative_sub_workflow.png)
 
-The Collaborative Sub-Workflow offers trade partners a means to manage
+The Collaborative Sub-Workflow enables trade partners a means to manage
```
Comment (shannynalayna) [inline text/0025-purchase-order.md]: I think here it sounds kind of wonky with "a means" , if using "enables" it sounds better without "a means"

File: text/0025-purchase-order.md [hunk 2]
```diff
@@ -309,31 +308,30 @@ Beginning state. An organization with po::draft permissions created a version
...
 further modification.
 
-Composed state. Both organizations are in agreement on the contents of the
-version. The buying organization retains the decision to issue a formal
+Composed state. Both organizations have accepted the contents of the
+version. The buying organization may decide to issue a formal
 purchase order based on the drafted contents.
 
-Declined state. An organization has declined the version content. An
+Declined state. An organization has declined the version content. Any
```
Comment (shannynalayna) [inline text/0025-purchase-order.md]: Hm, the use of "any" here makes it sound as though no permissions exist for that action

File: text/0025-purchase-order.md [hunk 3]
```diff
@@ -262,23 +263,22 @@ allow.
...
-seller’s system. The version is represented within the Confirmed state within
-the Purchase Order Sub-Workflow and takes the place of a previously accepted
-purchase order (if applicable). A buyer is permitted to cancel an accepted
-purchase order so long as a goods receipt or invoice has yet to be generated.
-This logic lives outside the scope of Grid at this time and is a concern of
-the Grid Integration component. A seller may take no further action on this
+Accepted state. A seller has reviewed and accepted the version in full. A separate sales
+order entity (out of scope of Grid Purchase Order) may be created within the
+seller’s system. The version is listed in the purchase order's
+`accepted_version` field, prelacing a previously accepted
```
Comment (shannynalayna) [inline text/0025-purchase-order.md]: is "prelacing" a word I don't know or should this be "replacing"

**Review by shannynalayna — APPROVED** [review]
Looks good to me, nice work

**Review by shannynalayna — APPROVED** [review]
Proto numbering looks good

**Review by vaporos — APPROVED** [review]

**Pull request merged**